Infrastructure/stack/igw.tf

```diff
@@ -1,7 +1,6 @@
 # Module for Internet Gateway
 module "harshvardhan-igw" {
```

Comment:
IGW is used in VPC so, you can include this in vpc.tf file, no need to create a separate file